src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -51,7 +63,9 @@ const InfoBox: UISchemaElementComponent<{
         {...({ 'data-se': dataSe })}
         className={`infobox-${messageClass.toLowerCase()}`}
       >
-        {message}
+        { title && <AlertTitle>{title}</AlertTitle> }
+        { message }
+        { bullets?.length > 0 && <List uischema={bulletList} /> }
```

Comment:
If the changes are made to the type (in schema), you can then just check the `listOptions` value for truthiness to display the List element. 

i.e. 
```
{ listOptions && <List uischema={{type: 'List', options={listOptions}}} /> }
```